site/cypress/e2e/test_forum.spec.js

```diff
@@ -0,0 +1,82 @@
+const title1 = 'Cypress Title 1 Cypress';
+const title2 = 'Cypress Title 2 Cypress';
+const title3 = 'Cypress Title 3 Cypress';
+const content1 = 'Cypress Content 1 Cypress';
+const content2 = 'Cypress Content 2 Cypress';
+const content3 = 'Cypress Content 3 Cypress';
+const reply1 = 'Cypress Reply 1 Cypress';
+const reply2 = 'Cypress Reply 2 Cypress';
+const reply3 = 'Cypress Reply 3 Cypress';
+const merged1 = 'Merged Thread Title: '.concat(title3, '\n\n', content3);
+const merged2 = 'Merged Thread Title: '.concat(title2, '\n\n', content2);
+
+const createThread = (title, content, category) => {
+    // Add more to tests for uploading attachments
+    cy.get('[title="Create Thread"]').click();
+    cy.get('#title').type(title);
+    cy.get('.thread_post_content').type(content);
+    cy.get('.cat-buttons').contains(category).click();
+    cy.get('[name="post"]').click();
+    cy.get('.flex-row > .thread-left-cont').should('contain', title);
+};
+
+const replyToThread = (title, reply) => {
+    cy.get('.thread-left-cont > .thread-list-item').contains(title).click();
+    cy.get('.create-post-head').should('contain', title);
+    cy.get('#reply_box_2').type(reply);
+    cy.get('[value="Submit Reply to All"]').click();
+    cy.get('#posts_list').should('contain', reply);
+};
+
+const mergeThreads = (fromThread, toThread, mergedContent) => {
+    // Add more to tests for uploading attachments
+    cy.get('.thread-left-cont > .thread-list-item').contains(fromThread).click({ force: true });
+    cy.get('[title="Merge Thread Into Another Thread"]').click();
+    cy.get('.chosen-single > span').click();
+    cy.wait(500);
+    cy.get('.active-result').contains(toThread).click({ force: true });
+    cy.get('#merge-threads > form > .popup-box > .popup-window > .form-body > .form-buttons > .form-button-container > .btn-primary').click({ force: true });
```

Comment:
Is there any better way to target this?  It just seems a bit fragile if someone changes the underlying HTML in the future.